recipes/django-cognito-jwt/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "django-cognito-jwt" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6e9dfb79b262c338a493b71f81811621f88fb0af88d4b86e896252a9c00c81bf
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - django >=1.11
+    - cryptography
+    - djangorestframework
+    - pyjwt
+    - requests
+
+test:
+  imports:
+    - django_cognito_jwt
+
+about:
+  home: https://github.com/labd/django-cognito-jwt
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Can you open a PR helping them address the fact that they aren't shipping their license.

Reply:
ANd cross reference the PR here.

Reply:
https://github.com/labd/django-cognito-jwt/issues/31